dev_guide/deployments/advanced_deployment_strategies.adoc

```diff
@@ -28,51 +30,49 @@ endif::[]
 ifdef::openshift-enterprise,openshift-dedicated,openshift-online[]
 Blue-green deployments
 endif::[]
-involve running two versions of an application at the same time and
-moving production traffic from the old, green, version to the new, blue, version.
-You can use a xref:deployment_strategies.adoc#rolling-strategy[rolling strategy]
-or switch services in a route.
-
-[[advanced-deployment-strategies-when-to-use-blue-green-deployment]]
-=== When to Use a Blue-Green Deployment
+involve running two versions of an application at the same time and moving
+traffic from the in-production version (the green version) to the newer version
+(the blue version). You can use a
+xref:../../dev_guide/deployments/deployment_strategies.adoc#rolling-strategy[rolling
+strategy] or switch services in a route.
 
-Use a xref:dev_guide/deployments/deployment_strategies.adoc#[blue-green deployment]
-when you want to switch users to a new, blue, version of your application in
-a production, green, environment.
+[NOTE]
+====
+Since many applications depend on persistent data, you will need to have an
+application that supports xref:n1-compatibility[N-1 compatibility] if you share
+a database, or implement a live data migration between your database, store, or
+disk if you choose to create two copies of your data layer.
```

Comment:
This paragraph is not clear to me. "you need to either:
* have an application that supports n-1
*implement live migration
??